packages/gen-ai-hub/src/core.ts

```diff
@@ -0,0 +1,71 @@
+import { DeploymentApi, AiDeployment } from '@sap-ai-sdk/ai-core';
```

Comment:
I guess putting it into this package can't be prevented 😬 
Perhaps we can call it something else than "core" though, to avoid the naming conflict.
Perhaps a `utils` subdirectory, and this is `resolver.ts`?